### PR TITLE
DAH-1153 events support markdown; clean up style for readability

### PR DESCRIPTION
**Pull request by kramduckner**
[DAH-1153](https://sfgovdt.jira.com/jira/software/c/projects/DAH/boards/122?modal=detail&selectedIssue=DAH-1153)

**Review Instructions**
Compare branch-deployed netlify (with updates) and https://dahlia-full.herokuapp.com/listings/a0W4U00000KRHeEUAX?react=true (without updates) and check the following items:

- There's only one information sessions containing two events
Screen Shot 2022-07-14 at 7 49 18 PM
- There's an underline in the section titles Open Houses and Information Sessions
 and There's extra vertical margin between events within a section, e.g. July 16th and July 21st

Screen Shot 2022-07-13 at 1 44 47 PM

- Markup isn't showing up in the content within Open Houses and Information Sessions
Screen Shot 2022-07-13 at 1 03 48 PM
- Click around app and make sure markup isn't showing up on any of the Open Houses or Information Sessions within focused listings
- Events without dates don't show Date. The event will still show up though. Users are using a date-less event for static events, like virtual tours. Here's an example on this particular listing. 
Screen Shot 2022-07-21 at 12 44 09 PM

Compare branch-deployed netlify (with updates) and https://dahlia-full.herokuapp.com/listings/listings/a0W4U00000KnHO6UAN?react=true (without updates) and check the following items:
- Open Houses are showing up. Feel free to click around other listings with completed lotteries that have open houses and ensure that open houses are displaying.

### CODE REVIEW

**Comment by christine-sfg** [comment]
Events component looks so much better and is more readable! Not done with review yet, but wanted to say great job!!

File: app/javascript/__tests__/components/__snapshots__/OpenHouses.test.tsx.snap [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    Open Houses
+  </h4>
+  <div
+    className="false"
+  >
+    <p
+      className="text text-gray-800 pb-2 flex justify-between items-center"
+    >
+      <span
+        className="inline-block text-tiny uppercase"
```
Comment (christine-sfg) [inline app/javascript/__tests__/components/__snapshots__/OpenHouses.test.tsx.snap:19]: any ideas on how best to not make the dates all uppercase?

**Review by christine-sfg — APPROVED** [review]
lgtm! just one question which is outside the scope of this pr

File: app/javascript/__tests__/components/__snapshots__/OpenHouses.test.tsx.snap (hunk 1 above)
Comment (kramduckner) [reply app/javascript/__tests__/components/__snapshots__/OpenHouses.test.tsx.snap:19]: yup! There's a PR in bloom for customization of the component with that uppercase field. Whatever strategy gets merged will support title-casing that date.

Comment (christine-sfg) [reply app/javascript/__tests__/components/__snapshots__/OpenHouses.test.tsx.snap:19]: awesome, thanks!

**Pull request merged**